packages/ui/brand/src/components/ComposerLogo/ComposerLogo.tsx

```diff
@@ -2,14 +2,16 @@
 // Copyright 2022 DXOS.org
 //
 
-import '@dxosTheme';
-
+import * as d3 from 'd3';
```

Comment:
Assets generally should not be animated with such a heavy library, though I know this is just something you’re experimenting with.

Reply:
yes currently experimental. but noted.